frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -78,6 +79,7 @@ export default function QueryVisualization(props) {
             maxTableRows={maxTableRows}
             className={CS.spread}
             onUpdateWarnings={setWarnings}
+            onUpdateVisualizationSettings={onUpdateVisualizationSettings}
```

Comment:
We need to pass `onUpdateVisualizationSettings` from the SDK's `QueryVisualization` component to `Visualization.jsx`